.github/workflows/eerelease.yml

```diff
@@ -0,0 +1,124 @@
+name: Release Ansible Execution Environment
+on:
+  workflow_dispatch:
+    inputs:
+      ee-name:
+        description: >-
+          EE name if it is community-ee-base or community-ee-minimal. Example: community-ee-base
+        required: true
+      ansible-core-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+      ee-version:
+        description: >-
+          EE Release Version. Example: 1
+        required: true
+      tag:
+        description: >-
+          Whether to tag the image as latest.
+        type: boolean
+        required: false
+        default: false
+
+env:
+  EE_VERSION: ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+
+jobs:
+  build:
+    name: Build Execution Environment ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+      packages: write
+
+    steps:
+      - name: Check out images
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/images
+          ref: main
+          path: image
+
+      - name: Check out eercheck repo for testing
+        uses: actions/checkout@v4
+        with:
+          repository: anweshadas/eercheck
+          ref: main
+          path: eercheck
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install ansible-builder
+          python3 -m pip install setuptools
+          sudo apt install podman --yes
+          pwd
+          ls -l
+
+      - name: Pull the latest fedora image
+        run: |
+          podman pull fedora:latest
+
+      - name: Build Community EE image
+        working-directory: image/execution-environments/${{ inputs.ee-name }}
+        run: |
+          ansible-builder build --tag ghcr.io/ansible-community/${{ inputs.ee-name }}:${{ env.EE_VERSION }}
+
+      - name: Find the id of the latest EE image
+        run: |
+          echo "IMAGE_ID=$(podman images -q ${{ inputs.ee-name }}:${{ env.EE_VERSION }})">>$GITHUB_ENV
+
+      - name: Test the image with eercheck
+        working-directory: eercheck
+        run: |
+          pwd
+          ls -l
+          python3 -m venv .venv
+          source .venv/bin/activate
+          python -m pip install -r requirements.txt
+          sudo systemctl start podman.socket
+          ./containertest.py ${{ env.IMAGE_ID }}
+          echo "STATUS_CODE=$?">>$GITHUB_ENV
+
+      - name: Checking the status code
+        run: |
+          if test ${{ env.STATUS_CODE }}!= 0; then
```

Comment:
I don't think `env.STATUS_CODE` exists. You need to give the previous task an ID, and refer to the output variables of that task (via its ID).

Reply:
I think you are asking for [this option](https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs#example-defining-outputs-for-a-job). I have followed this [one](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-writing-an-environment-variable-to-github_env) I am ready to update to the option you suggested. Is there any advantage of the one to the other?

Reply:
Ah, I overlooked the `_ENV`...